pool/src/test/pool_test.rs

```diff
@@ -528,13 +528,13 @@ fn deposit() {
         assert_eq!(token.balance(&user), initial_balance);
 
         let deposit_amount = 1_000_0;
-        let liq_index = RATE_DENOMINATOR + i * 100_000_000;
+        let liq_index = FixedI128::DENOMINATOR + i * 100_000_000;
```

Comment:
I think it's better to keep convention and use `FixedI128::ONE.into_inner()` like it's done in reserve_config.rs method